scripts/pcds_conda

```diff
@@ -30,72 +30,72 @@ CONDA_ROOT="${PYPS_ROOT}/conda"
+    first_bug="$(echo "${first_ver}" | cut -d . -f 3)"
+    second_maj="$(echo "${second_ver}" | cut -d . -f 1)"
+    second_min="$(echo "${second_ver}" | cut -d . -f 2)"
+    second_bug="$(echo "${second_ver}" | cut -d . -f 3)"
+    if [[ ${first_maj} -gt ${second_maj} ]]; then
         return 0
-      else
+    elif [[ ${first_maj} -lt ${second_maj} ]]; then
         return 1
-      fi
+    else
+        if [[ ${first_min} -gt ${second_min} ]]; then
+            return 0
+        elif [[ ${first_min} -lt ${second_min} ]]; then
+            return 1
+        else
+            if [[ ${first_bug} -ge ${second_bug} ]]; then
+                return 0
+            else
+                return 1
+            fi
+        fi
     fi
-  fi
 }
 
-
-if [[ -z "${PCDS_CONDA_VER}" ]]; then
-  # If env var unset, then default to latest
-  BASE_ENV="${CONDA_ROOT}/py39"
-  PYTHON_VER="3.9"
-  PCDS_CONDA_VER="$(cat "${BASE_ENV}/latest_env")"
+if [[ -z ${PCDS_CONDA_VER} ]]; then
+    # If env var unset, then default to latest
+    BASE_ENV="${CONDA_ROOT}/py39"
+    PYTHON_VER="3.9"
+    PCDS_CONDA_VER="$(cat "${BASE_ENV}/latest_env")"
 else
-  # Figure out which base to use
-  if [[ -z "${PCDS_CONDA_VER##*-*}" ]]; then
-    # We had a dash, there must be a prefix in the name
-    SERIES="$(echo "${PCDS_CONDA_VER}" | cut -d - -f 1)"
-    VER_NUM="$(echo "${PCDS_CONDA_VER}" | cut -d - -f 2)"
-  else
-    # We didn't have a dash or a prefix, assume it is pcds
-    SERIES="pcds"
-    VER_NUM="${PCDS_CONDA_VER}"
-    PCDS_CONDA_VER="pcds-${PCDS_CONDA_VER}"
-  fi
-  # Figure out where the base environment is
-  # Base environment should be updated with every new python version
-  # But in practice this can be skipped
-  if [[ "${SERIES}" == "pcds" ]]; then
-    if version_ge "${VER_NUM}" "${FIRST_PCDS_PY39}"; then
-      BASE_ENV="${CONDA_ROOT}/py39"
-      PYTHON_VER="3.9"
-    elif version_ge "${VER_NUM}" "${FIRST_PCDS_PY38}"; then
-      BASE_ENV="${CONDA_ROOT}/py36"
-      PYTHON_VER="3.8"
+    # Figure out which base to use
+    if [[ -z ${PCDS_CONDA_VER##*-*} ]]; then
+        # We had a dash, there must be a prefix in the name
+        SERIES="$(echo "${PCDS_CONDA_VER}" | cut -d - -f 1)"
+        VER_NUM="$(echo "${PCDS_CONDA_VER}" | cut -d - -f 2)"
     else
-      BASE_ENV="${CONDA_ROOT}/py36"
-      PYTHON_VER="3.6"
+        # We didn't have a dash or a prefix, assume it is pcds
+        SERIES="pcds"
+        VER_NUM="${PCDS_CONDA_VER}"
+        PCDS_CONDA_VER="pcds-${PCDS_CONDA_VER}"
+    fi
+    # Figure out where the base environment is
+    # Base environment should be updated with every new python version
+    # But in practice this can be skipped
+    if [[ ${SERIES} == "pcds" ]]; then
+        if version_ge "${VER_NUM}" "${FIRST_PCDS_PY39}"; then
+            BASE_ENV="${CONDA_ROOT}/py39"
+            PYTHON_VER="3.9"
+        elif version_ge "${VER_NUM}" "${FIRST_PCDS_PY38}"; then
+            BASE_ENV="${CONDA_ROOT}/py36"
```

Comment:
Should this be "py38"?  If we're using `PYTHON_VER="3.8"`?

Reply:
There is no py38 folder in the conda root

Reply:
So there's just py38 environments in the py36 folder? ha